### ISSUE TITLE
Apr 2020 PythonPune Newsletter Collection Box

### ISSUE THREAD

**Issue by raukadah** [pythonpune/meetup-talks#90]
Let's use this issue to collect python related news, new package updates and events happening around Pune. We can compile and publish it during **April PythonPune** meetup. 

## How to add news:
Comment on this issue with news link and news topic and also include the source from we got this. 

## How to add new package release:
Paste the link of new package release

## How to include events here:
If a new event is happening in Pune and wants to share in the newsletter, feel free to drop the event link with date and venue.

**Comment by raukadah** [comment]
New pip resolver to roll out this year  - http://pyfound.blogspot.com/2020/03/new-pip-resolver-to-roll-out-this-year.html

**Comment by raukadah** [comment]
How long did it take you to learn Python? - https://nedbatchelder.com/blog/202003/how_long_did_it_take_you_to_learn_python.html

**Comment by raukadah** [comment]
EuroPython 2020: Online conference from July 23-26 - https://blog.europython.eu/post/614102095419850752/europython-2020-online-conference-from-july-23-26

**Comment by raukadah** [comment]
What the heck is pyproject.toml? - https://snarky.ca/what-the-heck-is-pyproject-toml/

**Comment by Pradhvan** [comment]
@raukadah can we add a line below this 

> Let's use this issue to collect python related news, new package updates and events happening around Pune. We can compile and publish it during April PythonPune meetup.

stating whoever submits an article can submit a tl;dr version(one or two-line summary) just below their submitted article so that it's easy for the presenter of the newsletter at the meetup

**Comment by answerquest** [comment]
Automating GIS-processes by University of Helsinki, Finland  
https://automating-gis-processes.github.io/site/index.html  
Nice linear set of lessons complete with workable data and code on python + GIS.  
They also have another course for Python beginners: https://geo-python.github.io/site/

**Comment by bhavin192** [comment]
Post about the streaming setup we use during our meetup, 
https://geeksocket.in/posts/live-stream-meetup-event/

TL;DR: We use Jitsi Meet to host a meeting between two folks from the organizers and one presenter. This meeting is then live streamed to YouTube using Jitsi Meet itself or using OBS Studio. This post takes you through all the details of the setup.

**Comment by raukadah** [comment]
Python: Myths about Indentation - https://web.archive.org/web/20070922223915/http://www.secnetix.de/~olli/Python/block_indentation.hawk
TLDR: It talks about how python identifies between tabs and spaces, and parses it and what is the significance of it.

**Comment by raukadah** [comment]
Testing Brython apps using PyTest and Selenium http://jonathan.temno.eu/blog/testing_brython_selenium.html
TLDR: Brpython is a A Python 3 implementation for client-side web programming, designed to replace JavaScript and use python3 syntax in type of text/python. How selenium is used for testing the same.

**Comment by raukadah** [comment]
The Composition Over Inheritance Principle https://python-patterns.guide/gang-of-four/composition-over-inheritance/
TLDR: By using logging module, how composition is followed over Inheritance Principle while designing class.

**Comment by raukadah** [comment]
[Python-Dev] [RELEASE] Python 2.7.18, the end of an era - https://mail.python.org/archives/list/python-dev@python.org/message/OFCIETIXLX34X7FVK5B5WPZH22HXV342/
TLDR: it is the last release of Python-2.7

**Comment by raukadah** [comment]
Python's AsyncIO series by Łukasz Langa:
* Learn Python's AsyncIO #1 - https://www.youtube.com/watch?v=Xbl7XjFYsN4
* Learn Python's AsyncIO #2 - https://www.youtube.com/watch?v=E7Yn5biBZ58

**Comment by raukadah** [comment]
Ansible 101 by Jeff Geerling - YouTube streaming series - https://www.jeffgeerling.com/blog/2020/ansible-101-jeff-geerling-youtube-streaming-series

**Comment by raukadah** [comment]
US PyCon 2020 went virtual. Here is the videos: https://www.youtube.com/channel/UCMjMBMGt0WJQLeluw6qNJuA

**Comment by raukadah** [comment]
CFP for PyCon India 2020 is open now: https://in.pycon.org/cfp/2020/proposals/

**Comment by digitronik** [comment]
https://github.com/pythonpune/meetup-talks/pull/95
